packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -167,6 +189,10 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
         await this.webCallingService.registerWebCallingLine();
       }
 
+      this.webCallingService.setLoginOption(data.loginOption);
+      this.incomingTaskListener();
+      this.taskManager.registerIncomingCallEvent();
```

Comment:
should we switch off these listeners if we go into the catch block?

Reply:
The listeners can be enabled after the login response as discussed.

Reply:
done